### ISSUE TITLE
Autocomplete fails for any submodules

### ISSUE THREAD

**Issue by dburian** [tensorflow/tensorflow#59597]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Bug

### Have you reproduced the bug with TF nightly?

Yes

### Source

source

### Tensorflow Version

2.11.0 (v2.11.0-rc2-17-gd5b57ca93e5 ), 2.13.0-dev20230207 (v1.12.1-88998-gf03d41b6f8b)

### Custom Code

No

### OS Platform and Distribution

Linux Manjaro

### Python version

3.10

### Current Behaviour?

```shell
Typing any of the following does not offer any auto-completion:

import tensorflow as tf

tf.keras.<no-auto-completion>
tf.data.<no-auto-completion>
tf.estimator.<no-auto-completion>
tf.math.<no-auto-completion>
tf.lite.<no-auto-completion>


I would expect auto-completion to have some kind of knowledge about the submodules.
```

### Standalone code to reproduce the issue

```shell
import tensorflow as tf

tf.keras.<no-auto-completion>
tf.data.<no-auto-completion>
tf.estimator.<no-auto-completion>
tf.math.<no-auto-completion>
tf.lite.<no-auto-completion>
```

### Relevant log output

```shell
Using pyright version 1.1.292.
```
</details>

**Comment by pjpratik** [comment]
@dburian Thanks for reporting the issue.

I have tried in google colab and the auto complete suggestions are working fine. Please refer the below screenshot.

![image](https://user-images.githubusercontent.com/118897289/217450012-5a1647ca-dc38-465b-ad29-9ea8286e88f4.png)

I have tried with other modules as well and observed the same behaviour. Thanks!

**Comment by dburian** [comment]
That's great, yet I experience the lack of auto-complete with **pyright**, not **colab**. I posted the issue to see the possible solutions (either from others or the developers).

**Comment by pjpratik** [comment]
@dburian Thanks for the clarification.

@sushreebarsa  Could you please look into this.  Thank you.

**Comment by sachinprasadhs** [comment]
Hi, Thanks for opening the issue, we are aware of this issue in some of the IDE's which users have reported. 
We are working on the issue, for any progress regarding the similar issue, refer the threads in the issue here https://github.com/tensorflow/tensorflow/issues/56231, https://github.com/tensorflow/tensorflow/issues/36798

**Comment by alkatar21** [comment]
@dburian @sachinprasadhs  I think the Issue here will probably solved with https://github.com/microsoft/pylance-release/issues/3937? Which has the start of https://github.com/python/typeshed/issues/7144 as an underlying reason.

I mean tensorflow has other autocompletion problems, but at least the apearance of nothing working with pyright version 1.1.292 is probably caused by that.

**Comment by dburian** [comment]
Thanks a lot @alkatar21. Indeed in my case, the lack of auto-completion was caused by the (now incomplete) type stubs shipped with pyright, which pyright prefers over code inspection. For anyone experiencing the same issues try following the issues linked in alkatar's comment if you like to know the whys and whats.

TLDR; Try deleting 'tensorflow' directory under 'dist/typeshed-fallback/stubs'.
